src/tests/end-to-end/tests/content/a11y-content.test.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { contentPages } from '../../../../content';
+import { Browser } from '../../common/browser';
+import { launchBrowser } from '../../common/browser-factory';
+import { scanForAccessibilityIssues } from '../../common/scan-for-accessibility-issues';
+
+describe('A11Y for content pages', () => {
+    let browser: Browser;
+
+    beforeAll(async () => {
+        browser = await launchBrowser({ suppressFirstTimeDialog: true });
+    });
+
+    afterEach(async () => {
+        if (browser) {
+            await browser.closeAllPages();
+        }
+    });
+
+    afterAll(async () => {
+        if (browser) {
+            await browser.close();
+            browser = undefined;
+        }
+    });
+
+    const contentPaths = contentPages.allPaths();
+
+    it.each(contentPaths)('%s', async path => {
+        const content = await browser.newContentPage(path);
+
```

Comment:
you could also add test for hc theme

Reply:
out of scope as mention on **Notes for reviewers**

Reply:
@devaradhanm I had some more time today and end up adding tests for HC mode too.